diff --git a/tests/resources/app/coldbox/system/testing/VirtualApp.cfc b/tests/resources/app/coldbox/system/testing/VirtualApp.cfc
--- a/tests/resources/app/coldbox/system/testing/VirtualApp.cfc
+++ b/tests/resources/app/coldbox/system/testing/VirtualApp.cfc
@@ -94 +94 @@
-		return !isNull( application.cbController );
+		return application.keyExists( "cbController" ) && !isNull( application.cbController );
@@ -112 +112 @@
-		if ( !isNull( application.cbController ) ) {
+		if ( application.keyExists( "cbController" ) && !isNull( application.cbController ) ) {
diff --git a/tests/resources/app/coldbox/system/logging/config/LogBoxConfig.cfc b/tests/resources/app/coldbox/system/logging/config/LogBoxConfig.cfc
--- a/tests/resources/app/coldbox/system/logging/config/LogBoxConfig.cfc
+++ b/tests/resources/app/coldbox/system/logging/config/LogBoxConfig.cfc
@@ -64 +64 @@
-		if ( isNull( variables.utility ) ) {
+		if ( !variables.keyExists( "utility" ) || isNull( variables.utility ) ) {
diff --git a/tests/resources/app/coldbox/system/core/util/Util.cfc b/tests/resources/app/coldbox/system/core/util/Util.cfc
--- a/tests/resources/app/coldbox/system/core/util/Util.cfc
+++ b/tests/resources/app/coldbox/system/core/util/Util.cfc
@@ -296 +296 @@
-		if ( isNull( variables.mixerUtil ) ) {
+		if ( !variables.keyExists( "mixerUtil" ) || isNull( variables.mixerUtil ) ) {
diff --git a/tests/resources/app/coldbox/system/logging/Logger.cfc b/tests/resources/app/coldbox/system/logging/Logger.cfc
--- a/tests/resources/app/coldbox/system/logging/Logger.cfc
+++ b/tests/resources/app/coldbox/system/logging/Logger.cfc
@@ -399 +399 @@
-					if ( isNull( local.logEvent ) ) {
+					if ( !local.keyExists( "logEvent" ) || isNull( local.logEvent ) ) {
diff --git a/tests/resources/app/coldbox/system/core/delegates/Env.cfc b/tests/resources/app/coldbox/system/core/delegates/Env.cfc
--- a/tests/resources/app/coldbox/system/core/delegates/Env.cfc
+++ b/tests/resources/app/coldbox/system/core/delegates/Env.cfc
@@ -87 +87 @@
-		if ( isNull( variables.javaSystem ) ) {
+		if ( !variables.keyExists( "javaSystem" ) || isNull( variables.javaSystem ) ) {
diff --git a/tests/resources/app/coldbox/system/web/services/InterceptorService.cfc b/tests/resources/app/coldbox/system/web/services/InterceptorService.cfc
--- a/tests/resources/app/coldbox/system/web/services/InterceptorService.cfc
+++ b/tests/resources/app/coldbox/system/web/services/InterceptorService.cfc
@@ -194 +194 @@
-		if ( !isNull( arguments.interceptData ) ) {
+		if ( arguments.keyExists( "interceptData" ) && !isNull( arguments.interceptData ) ) {
diff --git a/tests/resources/app/coldbox/system/testing/BaseTestCase.cfc b/tests/resources/app/coldbox/system/testing/BaseTestCase.cfc
--- a/tests/resources/app/coldbox/system/testing/BaseTestCase.cfc
+++ b/tests/resources/app/coldbox/system/testing/BaseTestCase.cfc
@@ -162 +162 @@
-		if ( isNull( variables._ranBeforeAll ) ) {
+		if ( !variables.keyExists( "_ranBeforeAll" ) || isNull( variables._ranBeforeAll ) ) {
@@ -172 +172 @@
-		if ( isNull( variables._ranAfterAll ) ) {
+		if ( !variables.keyExists( "_ranAfterAll" ) || isNull( variables._ranAfterAll ) ) {
@@ -784 +784 @@
-		if ( !isNull( arguments.interceptData ) ) {
+		if ( arguments.keyExists( "interceptData" ) && !isNull( arguments.interceptData ) ) {
@@ -847 +847 @@
-		if ( isNull( variables.cbUtil ) ) {
+		if ( !variables.keyExists( "cbUtil" ) || isNull( variables.cbUtil ) ) {
@@ -859 +859 @@
-		if ( isNull( variables.env ) ) {
+		if ( !variables.keyExists( "env" ) || isNull( variables.env ) ) {
diff --git a/tests/resources/app/coldbox/system/ioc/Builder.cfc b/tests/resources/app/coldbox/system/ioc/Builder.cfc
--- a/tests/resources/app/coldbox/system/ioc/Builder.cfc
+++ b/tests/resources/app/coldbox/system/ioc/Builder.cfc
@@ -82 +82 @@
-		if ( isNull( variables.coldboxDSL ) ) {
+		if ( !variables.keyExists( "coldboxDSL" ) || isNull( variables.coldboxDSL ) ) {
@@ -94 +94 @@
-		if ( isNull( variables.cacheBoxDSL ) ) {
+		if ( !variables.keyExists( "cacheBoxDSL" ) || isNull( variables.cacheBoxDSL ) ) {
@@ -106 +106 @@
-		if ( isNull( variables.logBoxDSL ) ) {
+		if ( !variables.keyExists( "logBoxDSL" ) || isNull( variables.logBoxDSL ) ) {
diff --git a/tests/resources/app/coldbox/system/cache/store/ConcurrentStore.cfc b/tests/resources/app/coldbox/system/cache/store/ConcurrentStore.cfc
--- a/tests/resources/app/coldbox/system/cache/store/ConcurrentStore.cfc
+++ b/tests/resources/app/coldbox/system/cache/store/ConcurrentStore.cfc
@@ -261 +261 @@
-		if ( isNull( variables.collections ) ) {
+		if ( !variables.keyExists( "collections" ) || isNull( variables.collections ) ) {
diff --git a/tests/resources/app/coldbox/system/FrameworkSupertype.cfc b/tests/resources/app/coldbox/system/FrameworkSupertype.cfc
--- a/tests/resources/app/coldbox/system/FrameworkSupertype.cfc
+++ b/tests/resources/app/coldbox/system/FrameworkSupertype.cfc
@@ -621 +621 @@
-		if ( isNull( variables.asyncManager ) ) {
+		if ( !variables.keyExists( "asyncManager" ) || isNull( variables.asyncManager ) ) {
@@ -755 +755 @@
-		if ( isNull( variables.cbDateTimeHelper ) ) {
+		if ( !variables.keyExists( "cbDateTimeHelper" ) || isNull( variables.cbDateTimeHelper ) ) {
diff --git a/tests/resources/app/coldbox/system/core/util/Util.cfc b/tests/resources/app/coldbox/system/core/util/Util.cfc
--- a/tests/resources/app/coldbox/system/core/util/Util.cfc
+++ b/tests/resources/app/coldbox/system/core/util/Util.cfc
@@ -124 +124 @@
-		if ( isNull( variables.inetAddress ) ) {
+		if ( !variables.keyExists( "inetAddress" ) || isNull( variables.inetAddress ) ) {
diff --git a/tests/resources/app/coldbox/system/cache/config/CacheBoxConfig.cfc b/tests/resources/app/coldbox/system/cache/config/CacheBoxConfig.cfc
--- a/tests/resources/app/coldbox/system/cache/config/CacheBoxConfig.cfc
+++ b/tests/resources/app/coldbox/system/cache/config/CacheBoxConfig.cfc
@@ -107 +107 @@
-		if ( !isNull( cacheBoxDSL.logBoxConfig ) ) {
+		if ( structKeyExists( cacheBoxDSL, "logBoxConfig" ) && !isNull( cacheBoxDSL.logBoxConfig ) ) {
@@ -112 +112 @@
-		if ( !isNull( cacheBoxDSL.scopeRegistration ) ) {
+		if ( structKeyExists( cacheBoxDSL, "scopeRegistration" ) && !isNull( cacheBoxDSL.scopeRegistration ) ) {
@@ -117 +117 @@
-		if ( !isNull( cacheBoxDSL.caches ) ) {
+		if ( structKeyExists( cacheBoxDSL, "caches" ) && !isNull( cacheBoxDSL.caches ) ) {
@@ -125 +125 @@
-		if ( !isNull( cacheBoxDSL.listeners ) ) {
+		if ( structKeyExists( cacheBoxDSL, "listeners" ) && !isNull( cacheBoxDSL.listeners ) ) {
